Integrations/SyslogSender/SyslogSender.py

```diff
@@ -0,0 +1,278 @@
+        for mirror in mirrors:
+            if not mirror['mirrored']:
+                demisto.info('Mirroring: {}'.format(mirror['investigation_id']))
+                mirror = mirrors.pop(mirrors.index(mirror))
+                investigation_id = mirror['investigation_id']
+                mirror_type = mirror['mirror_type']
+                demisto.mirrorInvestigation(investigation_id, '{}:{}'.format(mirror_type, 'FromDemisto'), False)
+                mirror['mirrored'] = True
+                mirrors.append(mirror)
+
+                demisto.setIntegrationContext({'mirrors': json.dumps(mirrors)})
+
+
+def mirror_investigation():
+    """
+    Update the integration context with a new or existing mirror.
+    """
+    mirror_type = demisto.args().get('type', 'all')
+
+    investigation = demisto.investigation()
+
+    if investigation.get('type') == PLAYGROUND_INVESTIGATION_TYPE:
+        return_error('Can not perform this action in playground.')
+
+    investigation_id = investigation.get('id')
+    integration_context = demisto.getIntegrationContext()
+    if 'mirrors' not in integration_context:
+        mirrors: list = []
+    else:
+        mirrors = json.loads(integration_context['mirrors'])
+
+    mirror_filter = list(filter(lambda m: m['investigation_id'] == investigation_id, mirrors))
+    if mirror_filter:
+        # Delete existing mirror
+        mirrors.pop(mirrors.index(mirror_filter[0]))
+    mirror = {
+        'investigation_id': investigation.get('id'),
+        'mirror_type': mirror_type,
+        'mirrored': False
+    }
+
+    mirrors.append(mirror)
+    demisto.setIntegrationContext({'mirrors': json.dumps(mirrors)})
+
+    demisto.results('Investigation mirrored successfully.')
+
+
+''' Syslog send command '''
+
+
+def syslog_send(manager: SyslogManager, min_severity: int):
+    """
+    Send a message to syslog
+    """
+    message = demisto.args().get('message', '')
+    entry = demisto.args().get('entry')
+    ignore_add_url = demisto.args().get('ignoreAddURL', False)
+    log_level = demisto.args().get('log_level')
+    severity = demisto.args().get('severity')  # From server
+    message_type = demisto.args().get('messageType', '')  # From server
+
+    if severity:
+        try:
+            severity = int(severity)
+        except Exception:
+            severity = None
+
+    if message_type == INCIDENT_OPENED and (severity is not None and severity < min_severity):
+        return
+
+    if not message:
+        message = ' '
+
+    message = message.replace('\n', ' ').replace('\r', ' ').replace('`', '')
+    investigation = demisto.investigation()
+    if investigation:
+        if entry:
+            message = '{}, {}'.format(entry, message)
+        message = '{}, {}'.format(investigation.get('id'), message)
+
+    if ignore_add_url and isinstance(ignore_add_url, str):
+        ignore_add_url = bool(strtobool(ignore_add_url))
+    if not ignore_add_url:
```

Comment:
this is something generic we need for mirroring, lets add it to commonserver

Reply:
I'm not sure about that, the logic here is a little bit different(no line breaks), and and it's not that common...